internal/apis/kfd/v1alpha2/eks/delete/infrastructure.go

```diff
@@ -50,13 +51,23 @@ func NewInfrastructure(dryRun bool, workDir, binPath string, kfdManifest config.
 func (i *Infrastructure) Exec() error {
 	logrus.Info("Deleting infrastructure phase...")
 
+	timestamp := time.Now().Unix()
+
 	err := iox.CheckDirIsEmpty(i.OperationPhase.Path)
 	if err == nil {
 		logrus.Infof("infrastructure phase already executed, skipping...")
 
 		return nil
 	}
 
+	if err := i.tfRunner.Plan(timestamp, "-destroy"); err != nil {
+		return fmt.Errorf("error running terraform plan: %w", err)
```

Comment:
we are leaking terraform here, can we log something more relevant to the user?

Reply:
> Some minor suggestions on the modified code.
> 
> I've noticed some improvements that can be done to the help messages of the delete command, but they are out of the scope of this PR

The leaking tf messages were meant to be like the ones that are still in the code, so we can catch them all more easily then in a mixed scenario with tf e non tf messages in the codebase when working on https://github.com/sighupio/furyctl/issues/141